libs/frontend/feature-build/src/components/BuildDndOverlay.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <div class="build-dnd-overlay" :style="overlayStyle" />
+</template>
+
+<script lang="ts" setup>
+import { computed } from 'vue'
+import { runtimeContext } from '@pubstudio/frontend/util-runtime'
+import { useBuild } from '../lib/use-build'
+
+const hoverOutlineThickness = 3
+const outlineColor = '#F82389'
+
+const { editor } = useBuild()
+
+const overlayStyle = computed(() => {
```

Comment:
This is the function that calculates the offset&size of the D&D hover overlay.
It seems complicated, maybe there's a simpler way to do this. 😬

Reply:
It makes sense to me!